examples/tts/conf/es/fastpitch_align_44100.yaml

```diff
@@ -0,0 +1,222 @@
+# This config contains the default values for training grapheme Spanish FastPitch model with aligner using
+# 44.1KHz sampling rate. If you want to train model on other dataset, you can change config values according
+# to your dataset. Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+train_dataset: ???
+validation_datasets: ???
+sup_data_path: ???
+sup_data_types: ["align_prior_matrix", "pitch"]
+
+# Default values from librosa.pyin
+pitch_fmin: 65.40639132514966
+pitch_fmax: 2093.004522404789
+
+# Stats (per frame), train (these values depend on pitch_fmin and pitch_fmax)
+pitch_mean: 212.35873413085938
+pitch_std: 68.52806091308594
```

Comment:
[discussion] no need to change for this PR. we provided default values for pitch norm params in all yaml configs. I wonder if we should replace them with `???` just in case users forget to update with dataset-specific values?

Reply:
I do think it would be safer to leave them (and any other dataset specific configs like n_speakers) as ???

Reply:
Yeah, that's a good idea. Let's have a separate PR to set them to `???` across the board.

Reply:
ok. I will file a separate PR to replace all pitch norm params with `???`